config/webpack.config.js

```diff
@@ -16,7 +16,7 @@ const timestamp = new Date().getTime();
 
 const entryFiles = {
   burials: './src/js/burials/burials-entry.jsx',
-  'disability-benefits': './src/js/disability-benefits/disability-benefits-entry.jsx',
+  'claims-status': './src/js/claims-status/claims-status-entry.jsx',
```

Comment:
Can we do `claim-status` instead of `claims-status` (here and in other places)? I think that's how it's referred to most commonly, so it'll be easier.